pallets/sto/src/lib.rs

```diff
@@ -299,6 +300,11 @@ decl_module! {
             // Primary use is to calculate the blended price (offering_token_amount / cost).
             // Blended price must be <= to max_price or the investment will fail.
             let mut cost = T::Balance::from(0);
+
+            // Price is entered as a multiple of 1_000_000
+            // i.e. a price of 1 unit is 1_000_000
+            // a price of 1.5 units is 1_500_00
+            let price_divisor = T::Balance::from(1_000_000);
```

Comment:
We can use the `PerMil` type directly.

Reply:
I don't think the PerMil type is correct here. PerMil represents a number from [0, 1] whereas we are representing an amount of an asset, so Balance seems more correct?

Reply:
With `Permill` you'd use `x * (1 / divisor)` instead of `x / divisor` -- don't think either is more correct really?

Reply:
Yeah, you're right. Permill is limited to [0,1]. We'd have to use perbill if we want to go that route but just using `Balance` is probably best.